aclk/README.md

```diff
@@ -19,29 +19,16 @@ started with Cloud](https://learn.netdata.cloud/docs/cloud/get-started) guide or
 documentation](/claim/README.md).
 
 ## Data privacy
-
 [Data privacy](https://netdata.cloud/data-privacy/) is very important to us. We firmly believe that your data belongs to
-you. This is why **we don't store any metric data in Netdata Cloud**. 
-
-All the data that the user sees in the web browser when using Netdata Cloud, are actually streamed directly from the Netdata Agent to the Netdata Cloud dashboard. They pass through our systems, but they are not stored.
+you. This is why **we don't store any metric data in Netdata Cloud**.
 
-We do however store a limited number of *metadata* to be able to offer the stunning visualizations and advanced functionality of Netdata Cloud.
+All the data that you see in the web browser when using Netdata Cloud, is actually streamed directly from the Netdata Agent to the Netdata Cloud dashboard. 
+The data passes through our systems, but it isn't stored.
 
-### Metadata
+However, to be able to offer the stunning visualizations and advanced functionality of Netdata Cloud, it does store a limited number of _metadata_.
```

Comment:
Do we want to add a link to the new page here?

Reply:
@DShreve2 The following sentence does contain the link already. :)